apps/zbugs/server/notify.test.ts

```diff
@@ -0,0 +1,210 @@
+import {describe, it, expect} from 'vitest';
+import {type Schema} from '../shared/schema.ts';
+import {gatherRecipients} from './notify.ts';
+import type {ServerTransaction} from '@rocicorp/zero';
+import type postgres from 'postgres';
+
+// Mock database state
+const mockDB = {
```

Comment:
I was thinking something to test the SQL statement itself since that's where the core logic is that gathers recipients.

We have some `.pg-test.ts` files which will spin up and seed a postgres DB that you can hit.

So I'd:

1. Create a `email.pg-test.ts`
2. Seed with some data
3. Check that the big SQL statement `const recipientRows = await sql` returns what you expect for the `internal` issue case.

Reply:
example .pg-test.ts: https://github.com/rocicorp/mono/blob/51995101d0657519207f1c4695a8765b9016e07c/packages/zero-cache/src/services/change-source/pg/change-source.pg-test.ts#L56-L68